ui/app/templates/components/database-role-setting-form.hbs

```diff
@@ -1,5 +1,5 @@
 <div class="box is-sideless is-fullwidth is-marginless" data-test-role-settings-section>
-  <h3 class="title is-5">Role Settings</h3>
+  <h3 class="title is-5">Role settings</h3>
```

Comment:
A little ahead of myself and did a couple of `h3 class="title is-5"` that are sentence cased instead of titlelized like the `h1 class="title is-3"`